packaging/bootc/Containerfile.rhel

```diff
@@ -0,0 +1,22 @@
+FROM registry-proxy.engineering.redhat.com/rh-osbs/rhel9-rhel_bootc:rhel-9.4
+ADD etc etc
```

Comment:
Use `dnf copr enable redhat-et/flightctl-dev` instead of manually adding a repo file?

Reply:
we need to add the repo file for microshift anyway, and we'd avoid one command extra.